book/src/keymap.md

```diff
@@ -195,6 +195,7 @@ Jumps to various locations.
 | `g`   | Go to line number `<n>` else start of file       | `goto_file_start`          |
 | `e`   | Go to the end of the file                        | `goto_last_line`           |
 | `f`   | Go to files in the selection                     | `goto_file`                |
+| `u`   | Go to url in the selection                       | `goto_url`                 |
```

Comment:
It would be clearer if this especially mentioned that it will open the URL in a web browser.